dvc/dagascii.py

```diff
@@ -12,6 +16,44 @@
 from grandalf.routing import EdgeViewer
 from grandalf.routing import route_with_lines
 
+from dvc.utils import boxify
+
+
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_PAGER = "less"
+DEFAULT_PAGER_FORMATTED = "{} --chop-long-lines --clear-screen".format(
+    DEFAULT_PAGER
+)
+DVC_PAGER_VAR_NAME = "DVC_PAGER"
```

Comment:
This is a bit too much, just use `DVC_PAGER = "DVC_PAGER"` same as we do with other vars 🙂 Btw, we have `dvc/env.py` where we try to keep our env vars, lets put it there for now. Thinking about it, we have a debate about macros vs strings, so we might want to rethink that later 🙂